> I gather it's not shown due to a `.gitignore` rule

If I create a file `config.status-E`. then I see
```
% git status
On branch patch-18
Untracked files:
  (use "git add <file>..." to include in what will be committed)
	config.status-E

nothing added to commit but untracked files present (use "git add" to track)
% make test-git-no-uncommitted-changes 
./tools/test-git-no-uncommitted-changes
Error: the git repo has uncommitted changes:
?? config.status-E

make: *** [test-git-no-uncommitted-changes] Error 1
% echo $?
2
```
so it seems like it should be caught somewhere other than manually.

--- a/configure_wrapper
+++ b/configure_wrapper
@@ -15,7 +15,7 @@ cp conftest.py bak_conftest.py
 # Fix config.status to call ./configure instead of ./real_configure
 # so that --recheck also protects conftest.py
 if [[ "$OSTYPE" == "darwin"* ]]; then
-   sed -i '' -E "s|'./real_configure'|'./configure'|g" config.status
+   sed -i '' "s|'./real_configure'|'./configure'|g" config.status
 else
    sed -i "s|'./real_configure'|'./configure'|g" config.status
 fi

--- a/configure_wrapper
+++ b/configure_wrapper
@@ -14,4 +14,8 @@ cp conftest.py bak_conftest.py
 
 # Fix config.status to call ./configure instead of ./real_configure
 # so that --recheck also protects conftest.py
-sed -i '' -E "s|'./real_configure'|'./configure'|g" config.status
+if [[ "$OSTYPE" == "darwin"* ]]; then
+   sed -i '' -E "s|'./real_configure'|'./configure'|g" config.status
+else
+   sed -i "s|'./real_configure'|'./configure'|g" config.status
+fi

--- a/configure_wrapper
+++ b/configure_wrapper
@@ -14,8 +14,5 @@ cp conftest.py bak_conftest.py
 
 # Fix config.status to call ./configure instead of ./real_configure
 # so that --recheck also protects conftest.py
-if [[ "$OSTYPE" == "darwin"* ]]; then
-   sed -i '' "s|'./real_configure'|'./configure'|g" config.status
-else
-   sed -i "s|'./real_configure'|'./configure'|g" config.status
-fi
+sed -i.old "s|'./real_configure'|'./configure'|g" config.status
+rm -f config.status.old

